test/cases/aws.sh

```diff
@@ -216,7 +216,16 @@ fi
 
 greenprint "Pulling cloud-image-val container"
 
-CONTAINER_CLOUD_IMAGE_VAL="quay.io/cloudexperience/cloud-image-val-test:latest"
+REPO="${REPO:-os-build-composer}"
```

Comment:
Hm, I don't think this is correct, shouldn't it be `osbuild-composer`? Also if you plan on using the same script in CIV repo I guess it would be better to use some CI variable? `CI_PROJECT_NAME` might work. Or just set it in CI env vars or parse it from some project url...

Reply:
yes, The REPO variable is set in CIV env vars. I didn't saw any variable in `osbuild-composer` CI that could identify where the script is running so I just created that one. `CI_PROJECT_NAME` is a much better name though

Reply:
`CI_PROJECT_NAME` comes from https://docs.gitlab.com/ee/ci/variables/predefined_variables.html idk if the value is the actual name of the repo though, it sounds more like the project all the repos are in, I'd double check first.

Reply:
Ohh, I missunderstood you. Thanks, I'll double check